src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -2584,25 +2578,6 @@ public async Task SendAsync_RequestVersion20_ResponseVersion20IfHttp2Supported(U
             {
                 // It is generally expected that the test hosts will be trusted, so we don't register a validation
                 // callback in the usual case.
-                // 
-                // However, on our Debian 8 test machines, a combination of a server side TLS chain,
-                // the client chain processor, and the distribution's CA bundle results in an incomplete/untrusted
-                // certificate chain. See https://github.com/dotnet/corefx/issues/9244 for more details.
-                if (PlatformDetection.IsDebian8)
```

Comment:
The PR change description didn't explain why you got rid of this special code.  Why/how is this working better on Debian8?

Reply:
I look at linked #9244 and look pretty old. There may have been changes to OS or way how we build chains. 
We also changed default server we connect to so the chain may be different. So I did build and test run on Debian 8 and I did not see any problems. 
Because all of the above, I decided to remove the special handling.